### DISCUSSION TITLE
Unable to connect the postgres database

### DISCUSSION THREAD

**Discussion by X-SZM** [langgenius/dify#7435] [General]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Content

When I try to connect the  PostgreSQL databas using default settings via Navicat Premium Lite:
# PostgreSQL database configuration
DB_USERNAME=postgres
DB_PASSWORD=difyai123456
DB_HOST=localhost
DB_PORT=5432
DB_DATABASE=dify
error occurred:
FATEL：password authentication failed for user"postreg"
![屏幕截图 2024-08-20 143929](https://github.com/user-attachments/assets/f674c469-f2fa-44db-bb6c-86b4b7dd81b0)
Why is this happen and what should I do?

**Comment by crazywoola** [comment]
Run `docker inspect postgres` to find the correct host.
